vispy/visuals/volume.py

```diff
@@ -478,6 +485,7 @@
         vec4 color3 = vec4(0.0);  // final color
         vec3 dstep = 1.5 / u_shape;  // step to sample derivative
         gl_FragColor = vec4(0.0);
+        int surface_found = 0;
```

Comment:
Nitpick: maybe a bool is more clear for this purpose.

Reply:
Makes sense to me. Done!